docker-compose.yml

```diff
@@ -10,13 +10,19 @@ services:
     tty: true
     volumes:
       - ./.env:/home/node/app/.env
+    environment:
+      - REDIS_ENABLED=true
```

Comment:
can we keep a consisten syntax between services? the `relay` has: `- REDIS_ENABLED=true` and the `relay-ws` has `REDIS_ENABLED: true`.

I would recommend on using the latter, and also on quoting all the values.

Thanks!

Reply:
Done, can you check again.

Reply:
> quoting all the values.

Should boolean values be quoted too?

Reply:
when they represent environment values, **Yes!**